Applications/WebPortal/WebApp/library/source/pages/account_directory_page/account_directory_controller.tsx

```diff
@@ -1,6 +1,9 @@
+import * as Nexus from 'nexus';
 import * as React from 'react';
 import { DisplaySize } from '../..';
 import { LoadingPage } from '..';
+import { AccountDirectoryModel } from '.';

```

Comment:
Going forward we no longer allow ```import from '.'``` You can import from ```..``` or ```../..```, it's just ```.``` that is not allowed due to causing issues with circular dependencies.